core-bundle/src/Resources/contao/library/Contao/StringUtil.php

```diff
@@ -516,9 +516,9 @@ public static function toHtml5($strString)
 			'<u>'                                              => '<span style="text-decoration:underline">',
 			'</u>'                                             => '</span>',
 			' target="_self"'                                  => '',
-			' onclick="window.open(this.href); return false"'  => ' target="_blank"',
-			' onclick="window.open(this.href);return false"'   => ' target="_blank"',
-			' onclick="window.open(this.href); return false;"' => ' target="_blank"'
+			' onclick="window.open(this.href); return false"'  => ' target="_blank" rel="noreferrer noopener"',
+			' onclick="window.open(this.href);return false"'   => ' target="_blank" rel="noreferrer noopener"',
+			' onclick="window.open(this.href); return false;"' => ' target="_blank" rel="noreferrer noopener"'
```

Comment:
I am not sure if we can safely assume this replacement. What if someone has the following markup:

```html
<a href="…" onclick="window.open(this.href);return false" rel="noreferrer noopener">…</a>
```

They would end up with two `rel` attributes after the replacement.

Reply:
I think we should not change this method, but instead mark it as deprecated and stop using it in the core.